src/artemis/experiment_plans/full_grid_scan.py

```diff
@@ -38,7 +42,7 @@ def create_devices():
     i03.aperture_scatterguard().wait_for_connection()
 
 
-def wait_for_det_to_finish_moving(detector: DetectorMotion, timeout=2):
+def wait_for_det_to_finish_moving(detector: DetectorMotion, timeout=120):
```

Comment:
does it really need to be this long?

Reply:
Yep, the detector motion is slow, @neil-i03 said that 2mins is about right

Reply:
No, it's just how long until it will timeout. If the motion finishes earlier it will move on